### ISSUE TITLE
visual audit

### ISSUE THREAD

**Issue by duncangraham** [hivemindworks/Argo-Project-V0#108]
I need to go through the views and make sure we're using the correct templates. I'll probably find visualy bugs along the way that I'll update. As I make my way through the list I'll post updates on my progress, before and afters, and bugs.

## View checklist
- [x] account_menu.jade
- [x] account_page.jade
- [ ] charge_user_in_trip.jade
- [x] error.jade
- [x] index.jade
- [x] login.jade
- [ ] new_clientToken.jade
- [ ] new_payment_method.jade
- [x] signup.jade
- [ ] template_create.jade
- [ ] template_insert.jade
- [x] trip.jade
- [ ] trip_book.jade
- [x] trip_create.jade
- [ ] trip_edit.jade
- [ ] trip_list_charging.jade
- [ ] trip_settled.jade
- [ ] update_clientToken.jade
- [x] user_edit.jade
- [ ] user_in_trip.jade

**Comment by duncangraham** [comment]
got bogged down with datepicker work on the trip page. will continue pushing in the AM.

**Comment by duncangraham** [comment]
The datetimepicker doesn't look stellar, but it works for now. Moving on from the trip_create view.

<img width="1391" alt="screen shot 2015-08-12 at 2 21 07 pm" src="https://cloud.githubusercontent.com/assets/865298/9237038/74757800-40fd-11e5-87d8-f973f1ce30d6.png">
<img width="1391" alt="screen shot 2015-08-12 at 2 21 10 pm" src="https://cloud.githubusercontent.com/assets/865298/9237039/747ac65c-40fd-11e5-911a-5f141e233f6d.png">

**Comment by freindst** [comment]
It looks nice already. Can I give a suggestion? how about changing color of text in date-time picker to black? green color words make eyes tired.

**Comment by duncangraham** [comment]
thanks for the suggestion @freindst 

more progress

<img width="1391" alt="screen shot 2015-08-13 at 2 20 28 pm" src="https://cloud.githubusercontent.com/assets/865298/9261992/9da82492-41c6-11e5-84b5-13e361381270.png">
<img width="1391" alt="screen shot 2015-08-13 at 2 20 31 pm" src="https://cloud.githubusercontent.com/assets/865298/9261993/9da9b5b4-41c6-11e5-8163-cf761dfb3259.png">
<img width="1391" alt="screen shot 2015-08-13 at 2 21 02 pm" src="https://cloud.githubusercontent.com/assets/865298/9261991/9da8138a-41c6-11e5-8547-933a02319d76.png">

**Comment by markstm17** [comment]
I like @freindst suggestion. Doesn't need to be changed right now. But If I recall when I just made my trip it was green

It seems like the Organizer can delete and edit trip. Editing trip details will be vital for the organizer in the future without Argobus Admin approval, but for now is fine @duncangraham 


And @duncangraham the datetime picker didn't have the drop downs for the calendar and time when I just created my trip
